nautobot/ipam/api/serializers.py

```diff
@@ -265,6 +265,16 @@ def validate(self, data):
         if self.instance is None and not any([namespace, parent]):
             raise ValidationError({"__all__": "One of parent or namespace must be provided"})
 
+        # DRF does not call Model clean method by default; however, to avoid rewriting validation checks in Model clean, call clean method here.
```

Comment:
DRF doesn't call model clean by default, but Nautobot's `ValidatedModelSerializer` (parent of `NautobotModelSerializer` and therefore of this serializer) does do so. Probably a more correct fix here would be to call `super.validate(data)` at some point, rather than calling `instance.clean()` directly?